packages/nimble-components/src/spinner/tests/spinner.stories.ts

```diff
@@ -39,6 +39,12 @@ const metadata: Meta<SpinnerArgs> = {
         design: {
             artboardUrl:
                 'https://xd.adobe.com/view/33ffad4a-eb2c-4241-b8c5-ebfff1faf6f6-66ac/screen/dece308f-79e7-48ec-ab41-011f3376b49b/specs/'
+        },
+
+        // Spinner animation causes snapshot changes in chromatic
+        // See https://github.com/ni/nimble/issues/983
+        chromatic: {
```

Comment:
The storybook build for this change is causing every test to show as new while disabling the spinner test, this makes me nervous
![image](https://user-images.githubusercontent.com/1588923/216161868-af5dbc19-ff98-4e32-bd07-2b355fedf5a8.png)

Reply:
nvm, a newer build trigger seems to have resolved that